### PR TITLE
Port cc 10126

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Port PR https://github.com/openvinotoolkit/openvino/pull/10126
 - Extend CC compilation step for build custom OV

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: .ci/azure/windows_conditional_compilation.yml [hunk 1]
```diff
@@ -95,16 +109,27 @@ jobs:
...
+  - script: |
+      call "$(MSVS_VARS_PATH)" && cmake -G"Visual Studio 16 2019" -DVERBOSE_BUILD=ON -DENABLE_FASTER_BUILD=ON -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DENABLE_PROFILING_ITT=OFF -DSELECTIVE_BUILD=ON -DSELECTIVE_BUILD_STAT=$(BUILD_DIR)\*.csv -DCMAKE_C_COMPILER:PATH="$(MSVC_COMPILER_PATH)" -DCMAKE_CXX_COMPILER:PATH="$(MSVC_COMPILER_PATH)" $(REPO_DIR)
+    workingDirectory: $(BUILD_DIR_2)
+    displayName: 'CMake2'
 
-  - script: dir $(INSTALL_DIR) /s
-    displayName: 'List install files'
+  - script: cmake --build . --config $(BUILD_TYPE)
+    workingDirectory: $(BUILD_DIR_2)
+    displayName: 'Build WinCC ON'
```
Comment (ilya-lavrenov) [inline .ci/azure/windows_conditional_compilation.yml:132]: should we also add a step that BA can run this topology?

Comment (ilya-lavrenov) [reply .ci/azure/windows_conditional_compilation.yml:132]: BTW, Win CC now takes 100 mins. Can we turn off unused plugins and other functionality like frontends where CC does not make sence

Comment (ilyachur) [reply .ci/azure/windows_conditional_compilation.yml:167]: Disabled the last check due to the issue on CPU. 
This step should be enabled after the fix

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
@ilyachur thanks!

File: .ci/azure/windows_conditional_compilation.yml (hunk 1 above)
Comment (ilya-lavrenov) [reply .ci/azure/windows_conditional_compilation.yml:167]: > Disabled the last check due to the issue on CPU.

Do we have a ticket for this issue?

**Pull request merged**